registers/TUTORIAL.md

```diff
@@ -19,3 +19,45 @@ You can run it with
 By default it uses the ["register" register (a list of all
 registers)](https://register.register.gov.uk/) but you can point it at
 others with the `--register` flag.
+
+Step 2
+------
+
+Next, we are going to put the data into a Trillian log. The first step
+is run the log server and create a log.
+
+Build Trillian:
+
+```make trillian```
```

Comment:
Hiding the actual commands behind make targets feels like it defeats the purpose of this being a tutorial.

I prefer if the full commands were actually spelled out here.